src/main/java/chatterchicken/ui/Ui.java

```diff
@@ -0,0 +1,93 @@
+package chatterchicken.ui;
+
+import chatterchicken.task.Task;
+import java.util.ArrayList;
+
+public class Ui {
+
+    private static final String LINE = "\n    _____________________________________________________________________________\n";
+
+    private static final String INDENT = "      ";
+    private static final String INDENT_BIG = "        ";
```

Comment:
I like how you differentiate the indentations and make them constants, it makes reading the output messages much easier!